lib/notification_manager.rb

```diff
@@ -17,7 +17,8 @@ def error(e, message, callee = nil)
         log_exception(pubmed_logger, log_message, e)
       when Cap::AuthorsPoller, Cap::Client
         log_exception(cap_logger, log_message, e)
-      when WebOfScience::Client, WebOfScience::Harvester, WebOfScience::ProcessRecords, WebOfScience::Record
+      when WebOfScience::Client, WebOfScience::Contributions,
```

Comment:
Still literally no reason to keep using this bogus pattern.

Reply:
I can't tell if this is a general gripe that needs more discussion outside this PR or if this is asking for a change to this PR.  I'm assuming for the former for now.